otx/mpa/modules/models/heads/custom_atss_head.py

```diff
@@ -36,6 +36,15 @@ def __init__(
         self.bg_loss_weight = bg_loss_weight
         self.use_qfl = use_qfl
 
+    def _init_layers(self):
```

Comment:
Is this a function called from the mmdetection side?

Reply:
Yes it is in the latest mmdet's atss_head class